Fix build failure against network

src/Network/SSH/Client/LibSSH2.hs:40:51:
    Ambiguous occurrence `sClose'
    It could refer to either `Network.sClose',
                             imported from `Network' at src/Network/SSH/Client/LibSSH2.hs:25:1-14
                          or `Network.Socket.sClose',
                             imported from `Network.Socket' at src/Network/SSH/Client/LibSSH2.hs:27:1-21
diff --git a/src/Network/SSH/Client/LibSSH2.hs b/src/Network/SSH/Client/LibSSH2.hs
index 9377cc0..5a321b4 100644
--- a/src/Network/SSH/Client/LibSSH2.hs
+++ b/src/Network/SSH/Client/LibSSH2.hs
@@ -37,7 +37,7 @@ import Network.SSH.Client.LibSSH2.Foreign
 socketConnect :: String -> Int -> IO Socket
 socketConnect hostname port = do
     proto <- getProtocolNumber "tcp"
-    bracketOnError (socket AF_INET Stream proto) (sClose)
+    bracketOnError (socket AF_INET Stream proto) (Network.sClose)
             (\sock -> do
               he <- getHostByName hostname
               connect sock (SockAddrInet (fromIntegral port) (hostAddress he))
